### ISSUE TITLE
predict tile per tile with basic transfer learning

### ISSUE THREAD

**Issue by ldocao** [ldocao/owkin_breast_cancer#9]
following discussion in #6 

- [x] preprocess images from kaggle (resize, normalize)
- [ ] visualize loss: https://keras.io/visualization/
- [x] augment data
- [ ] fine tune deeper layers
- [ ] crop center

**Comment by ldocao** [comment]
If I get this error:

OMP: Error #15: Initializing libiomp5.dylib, but found libiomp5.dylib already initialized. OMP: Hint This means that multiple copies of the OpenMP runtime have been since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.

this is solved by installing: conda install nomkl

https://stackoverflow.com/questions/53648730/omp-error-15-initializing-libiomp5-dylib-but-found-libiomp5-dylib-already-in

**Comment by ldocao** [comment]
Transfer learning succeeded with adding only a FC layer after average pooling (included at download time of original inception_v3, ie use pooling='avg' keyword). The Dense layer has a single output, with "binary_crossentropy" as loss. We could check the following elements for improvement:

- change architecture (add other layers)
- change loss
- change optimizer

**Comment by ldocao** [comment]
some commits have been referred as #7 whereas it should actually belong to this issue

**Comment by ldocao** [comment]
I have a conflict between opencv and matplotlib , each time I want to use plt.plot()

libc++abi.dylib: terminating with uncaught exception of type NSException
Abort trap: 6

**Comment by ldocao** [comment]
There is a quota limit on GCP about the GPU, request sent
